### PR TITLE
rpcs3-dev: new package

### PR DESCRIPTION
**Pull request by matoro**

### CODE REVIEW

File: bucket/rpcs3-dev.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://rpcs3.net/",
+    "description": "Open-source Sony Playstation 3 emulator (development version)",
+    "license": "GPL2",
```
Comment (Calinou) [inline bucket/rpcs3-dev.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: bucket/rpcs3-dev.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://rpcs3.net/",
+    "description": "Open-source Sony Playstation 3 emulator (development version)",
```
Comment (Calinou) [inline bucket/rpcs3-dev.json]:
```suggestion
    "description": "Open-source Sony PlayStation 3 emulator (development version)",
```

**Comment by matoro** [comment]
@Calinou Done!

**Comment by Calinou** [comment]
Thanks!

**Comment by matoro** [comment]
@Calinou There seem to be a few issues with the manifest that are causing it to fail validation against the official schema and thus fail the CI builds.  Could you take a look?

**Comment by matoro** [comment]
Oops, nevermind - just realized I messed up the URL schema.  Fixed here:  https://github.com/Calinou/scoop-games/pull/89

**Pull request merged**